python/adios2/file_reader.py

```diff
@@ -12,8 +12,10 @@ class FileReader(Stream):
     def __repr__(self):
         return f"<adios.file named {self._io_name}>"
 
-    def __init__(self, path, comm=None, engine_type="BPStream", config_file=None):
-        super().__init__(path, "rra", comm, engine_type, config_file)
+    def __init__(self, path, comm=None, *, engine_type=None, config_file=None, io_name=None):
```

Comment:
should we move the `*` before comm?

Reply:
I was debating this. Named comm=comm argument looks more consistent for optional arguments but leaving it as a normal argument keeps it closer to the existing high-level API. Undecided.

Reply:
Decided to go with your suggestion.